mtags/src/main/scala/scala/meta/internal/pc/Keywords.scala

```diff
@@ -0,0 +1,139 @@
+package scala.meta.internal.pc
+
+import org.eclipse.{lsp4j => l}
+
+import scala.collection.mutable
+
+trait Keywords { this: MetalsGlobal =>
+
+  private[this] def blockDefinitions: List[String] =
+    List("def", "val", "var", "lazy val", "type")
+
+  private[this] def toplevelDefinitions: List[String] =
+    List("case class", "class", "trait", "object")
+
+  private[this] def definitionModifiers =
+    List("final", "private", "protected", "sealed trait",
+      "sealed abstract class", "abstract")
+
+  // Categories
+
+  private[this] def expressionKeywords: List[String] =
+    List("super", "this", "if", "for", "while", "do", "true", "false", "new",
```

Comment:
If we have try then maybe `catch` too? And we could consider apply as being useful to include

Reply:
Context-sensitive keywords like `catch`, `finally`, `yield` and `extends` can be left for a separate PR.

Reply:
apply is not a keyword though, so I would not include it here